### PR TITLE
Mac OS support added

### PR DESCRIPTION
**Pull request by kunjee17**
@jimbobbennett Hi,

here is mac os support. I am not Cocoa expert so there are some issues there. 

- two windows are getting opened 
- application windows is coming small so I need to maximize that 

Other than that it just work without any issue. 

And I m sorry if you need to change the name of application after this PR. :P 

PS: Please help me solve this issue before accepting it, if possible. Accepting and reject is totally up to you. I was just trying how fast I can come up with Mac support. I guess that is super fast.

### CODE REVIEW

File: SqueakyApp.Mac/AppDelegate.fs [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+open Foundation
+open AppKit
+open Xamarin.Forms
+open Xamarin.Forms.Platform.MacOS
+
+[<Register ("AppDelegate")>]
+type AppDelegate () =
+    inherit FormsApplicationDelegate ()
+
+    let mutable _window = new NSWindow()
```
Comment (jimbobbennett) [inline SqueakyApp.Mac/AppDelegate.fs]: Change this to be

`let mutable _window : NSWindow = null`

as part of the fix to remove the extra window.

**Comment by jimbobbennett** [comment]
Just added some commits to fix up the extra window. 

I also see the window looking pretty bad - white text on white. Do you see the same? I am on Mojave so could be related to that.

**Comment by jimbobbennett** [comment]
I'd like to merge this in. Can you also update the README.MD to include macOS and add the instructions on how someone could re-create the same thing, same as I have done for Linux.

**Comment by kunjee17** [comment]
@jimbobbennett Yeah sure. I can do that.

**Comment by kunjee17** [comment]
@jimbobbennett please do have a look. Now single window is coming. And I also have added screen shot and read me for the same. This was too much fun. I never thought that I will have so much fun making desktop application again.

**Pull request merged**